<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sprite Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="brushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>58</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="eraseButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>58</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="eyedropButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>58</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="negativeButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>58</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="redSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="colorLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>285</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QSlider" name="redSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="greenSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSlider" name="greenSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="blueSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSlider" name="blueSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="alphaSpinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSlider" name="alphaSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="animationSlider">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>89</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="animationExpandButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>90</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="canvas">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="animationLabel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="frameNumberLabel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>450</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame Number</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="frameNumberSpinBox">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>450</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="addFramePushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>480</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteFramePushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>510</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="BrushSizeLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Brush Size</string>
    </property>
   </widget>
   <widget class="QSlider" name="brushSizeHorizontalSlider">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="brushSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="gridLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="colorPreview">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="framePreview1">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>340</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="framePreview2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>240</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="framePreview3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="frameSelect2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>268</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="frameSelect3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>168</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuChange_Canvas_Size">
     <property name="title">
      <string>Resize Canvas</string>
     </property>
     <addaction name="separator"/>
     <addaction name="action1x1"/>
     <addaction name="action2x2"/>
     <addaction name="action4x4"/>
     <addaction name="action8x8"/>
     <addaction name="action16x16"/>
     <addaction name="action32x32"/>
     <addaction name="action64x64"/>
    </widget>
    <addaction name="menuChange_Canvas_Size"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action1x1">
   <property name="text">
    <string>1x1</string>
   </property>
  </action>
  <action name="action2x2">
   <property name="text">
    <string>2x2</string>
   </property>
  </action>
  <action name="action4x4">
   <property name="text">
    <string>4x4</string>
   </property>
  </action>
  <action name="action8x8">
   <property name="text">
    <string>8x8</string>
   </property>
  </action>
  <action name="action16x16">
   <property name="text">
    <string>16x16</string>
   </property>
  </action>
  <action name="action32x32">
   <property name="text">
    <string>32x32</string>
   </property>
  </action>
  <action name="action64x64">
   <property name="text">
    <string>64x64</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
